datahub-graphql-core/src/main/java/com/linkedin/datahub/graphql/resolvers/type/ResultsTypeResolver.java

```diff
@@ -0,0 +1,21 @@
+package com.linkedin.datahub.graphql.resolvers.type;
+
+import com.linkedin.datahub.graphql.generated.StringBox;
+
+import graphql.TypeResolutionEnvironment;
+import graphql.schema.GraphQLObjectType;
+import graphql.schema.TypeResolver;
+
+public class ResultsTypeResolver implements TypeResolver {
+
+    public static final String STRING_BOX = "StringBox";
+
+    @Override
+    public GraphQLObjectType getType(TypeResolutionEnvironment env) {
+        if (env.getObject() instanceof StringBox) {
```

Comment:
Any other types?

Reply:
As of now its just String - https://github.com/linkedin/datahub/blob/master/metadata-models/src/main/pegasus/com/linkedin/ml/metadata/ResultsType.pdl

It's made a Union type to expand in the future